client/app/routes/deis.js

```diff
@@ -0,0 +1,19 @@
+import Route from '@ember/routing/route';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default class DeisRoute extends Route.extend(AuthenticatedRouteMixin) {
+  authenticationRoute = '/';
+
+  async model(params) {
+    const DeisPackage = await this.store.findRecord('package', params.id, {
```

Comment:
this variable should start with an undercase: `deisPackage` 
This is a naming convention of javascript: to use camelCase. 
The first letter is always undercase, then the first letter of each segment is capitalized